cpp_linter/cli.py

```diff
@@ -358,6 +362,22 @@ class Args(UserDict):
     help="""Set to ``true`` to prevent Pull Request
 reviews from requesting or approving changes.""",
 )
+_parser_args[("-C", "--delete-comments")] = dict(
+    default="true",
+    type=lambda input: input.lower() == "true",
+    help="""Set to ``true`` to delete existing outdated/unused
+review comments, ``false`` to just set them to resolved.
+
+Defaults to ``%(default)s``.""",
+)
```

Comment:
The behavior when `--thread-comments` is set to 'true'. Why do we need a separate/new CLI option?

Reply:
This is a terminology difference (perhaps '--delete-review-comments' or something)
`--thread-comments` effects these comments:
![comment](https://github.com/user-attachments/assets/6d835410-6719-4aa4-9695-545c8121e666)
whereas this option effects these comments:
![image](https://github.com/user-attachments/assets/479b632d-43d6-4ab9-a58a-fc392aef322c)
Github differentiates them as the former are pull request comments and the latter are pull request review thread comments and get handled differently. The latter require the graphQL functionality in this PR to change

Reply:
If the new options arent clear then feel free to suggest better naming

Reply:
oh I see. Maybe a more explicit otion name then? like `--delete-review-comments` or `--reuse-review-comments`.

Reply:
Yep thats probably a good suggestion, Ive update the options in this PR accordingly